.github/workflows/e2e.yml

```diff
@@ -285,163 +282,35 @@ jobs:
         working-directory: test/e2e
         run: yarn install
 
-      - name: Build SDK
-        run: yarn build
-
-      - name: Build E2E Tests Library
-        working-directory: test/e2e
-        run: yarn build
-
-      - name: Package SDK
-        run: yalc publish
-
       - uses: actions/setup-node@v4
         if: ${{ matrix.rn-version == '0.65.3' }}
         with:
           node-version: 16
 
-      - name: Download Plain RN ${{ matrix.rn-version }} App
-        working-directory: test/react-native/versions
-        run: git clone $RN_DIFF_REPOSITORY --branch release/${{ matrix.rn-version }} --single-branch ${{ matrix.rn-version }}
-
-      - name: Add SDK to App
-        working-directory: test/react-native/versions/${{ matrix.rn-version }}/RnDiffApp
-        run: yalc add @sentry/react-native
-
-      - name: Install App JS Dependencies (yarn v1)
-        working-directory: test/react-native/versions/${{ matrix.rn-version }}/RnDiffApp
-        if: ${{ matrix.rn-version != '0.73.9' }}
-        run: |
-          yarn install
-
-      - name: Install App JS Dependencies (yarn v3)
-        working-directory: test/react-native/versions/${{ matrix.rn-version }}/RnDiffApp
-        if: ${{ matrix.rn-version == '0.73.9' }}
-        run: |
-          rm -f .yarnrc.yml # original yarnrc contains the exact yarn version which causes corepack to fail to install yarn v3
-          echo "nodeLinker: node-modules" > .yarnrc.yml # RN build script require dependencies to be present in node_modules
-          touch yarn.lock # yarn v3 won't install dependencies in a sub project without a yarn.lock file present
-          export YARN_ENABLE_IMMUTABLE_INSTALLS=false # yarn v3 run immutable install by default in CI
-          yarn install
-
-      - name: Add E2E Tests Library to App
-        working-directory: test/react-native/versions/${{ matrix.rn-version }}/RnDiffApp
-        run: yarn add ../../../../e2e
-
       - uses: ruby/setup-ruby@v1
         if: ${{ matrix.platform == 'ios' }}
         with:
-          working-directory: test/react-native/versions/${{ matrix.rn-version }}/RnDiffApp
```

Comment:
Without the working dir the `bundle install` won't run in the `testAppDirectory`. So likely the gems cache won't be working correctly.

Reply:
I'm not sure if I understand.

Reply:
I'm saying I've just made a change. let's see if the step works now in CI